Basic Sponsorship and Mentor Update

CharlieHammersley opened the pull request:
I changed the link for the sponsorship and removed inactive mentors, I might add another section for past mentors. Some of the text was changed including dates of mentors which may be incorrect but are mostly placeholders for now. In the future I will add more to the meet the team page and add the current and soon to be SMEs.

CharlieHammersley committed:
Update sponsor-us.tsx

Updated the sponsor link to the currently used sponsor link.

CharlieHammersley committed:
Update meet-the-mentors-page.tsx

Removed inactive mentors and updated dates.

CharlieHammersley committed:
Update meet-the-team.tsx

Updated some of the text for the About Us section.

CharlieHammersley committed:
Update sponsor-us.tsx

Removed the space in the link.

T99 commented:
@CharlieHammersley Hey! If you could merge `master` back into your branch, I'll get a feature deployment of this out so that we can test it.

CharlieHammersley committed:
Merge branch 'frc1711:master' into master

CharlieHammersley committed:
Update sponsor-us.tsx

Edited the sponsor-us page to include accurate and up to date information about the team.

CharlieHammersley committed:
Update SME text meet-the-leads-page.tsx

Updated the SME text boxes to reflect our current/soon to be SMEs. The images are not yet updated.

CharlieHammersley committed:
Uploaded Drive Team Photos

CharlieHammersley committed:
Rename IMG_6270.jpeg to milo-headshot-drive-team.jpeg

CharlieHammersley committed:
Rename IMG_6333.jpeg to kayden-headshot-drive-team.jpeg

CharlieHammersley committed:
Rename IMG_6281.jpeg to frannie-headshot-drive-team.jpeg

CharlieHammersley committed:
Rename IMG_6330.jpeg to oscar-headshot-drive-team.jpeg

CharlieHammersley committed:
Rename IMG_6319.jpeg to aviannah-headshot-drive-team.jpeg

CharlieHammersley committed:
Delete src/assets/images/aviannah-headshot-drive-team.jpeg

CharlieHammersley committed:
Delete src/assets/images/frannie-headshot-drive-team.jpeg

CharlieHammersley committed:
Rename kayden-headshot-drive-team.jpeg to dt-kayden

CharlieHammersley committed:
Delete src/assets/images/dt-kayden

CharlieHammersley committed:
Delete src/assets/images/oscar-headshot-drive-team.jpeg

CharlieHammersley committed:
Delete src/assets/images/milo-headshot-drive-team.jpeg

CharlieHammersley committed:
Rename unnamed.jpg to dt-group-2026.jpg

CharlieHammersley committed:
Reuploaded 2026 drive team photos

CharlieHammersley committed:
Create blank

CharlieHammersley committed:
Delete src/assets/SMES directory

CharlieHammersley committed:
Rename src/assets/images/aviannah-headshot.jpeg to src/assets/images/smes/aviannah-headshot.jpeg

CharlieHammersley committed:
Delete src/assets/images/smes directory

CharlieHammersley committed:
Delete src/assets/images/oscar-headshot.jpeg

CharlieHammersley committed:
Delete src/assets/images/frannie-headshot.jpeg

CharlieHammersley committed:
Delete src/assets/images/milo-headshot.jpeg

CharlieHammersley committed:
Delete src/assets/images/kayden-headshot.jpeg

CharlieHammersley committed:
Delete src/assets/images/drive-team-group-photo.jpg

CharlieHammersley committed:
Create blank

CharlieHammersley committed:
re-reupload of sme images

CharlieHammersley committed:
Delete src/assets/images/smes/blank

CharlieHammersley committed:
Updated meet-the-leads-page.tsx

Imported the new photos and updated each blurb.

CharlieHammersley committed:
Update sponsor-us.tsx

Updated link to open the Responder view instead of Edit.

CharlieHammersley committed:
Update home.tsx image

Updated the home image to a placeholder of the 2026 drive team. We can replace this later with the yearbook team picture or a banner photo.

CharlieHammersley committed:
Update meet-the-leads-page.tsx

final touches on SME blurbs
